sys/include/net/gnrc/ipv6/blacklist.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
yes it was intentional, this module is pretty the same as @authmillenon's `gnrc_ipv6_whitelist`.
I just extended/changed her idea/implementation to blacklisting.